src/index.js

```diff
@@ -99,15 +99,15 @@ const extendCodeBlockWithSyntaxHighlighting = (settings) => {
 		);
 	};
 
-	const parseSelectedLines = (selectedLines) => {
-		const highlightedLines = new Set();
+	const parseSelectedLines = (highlightedLines) => {
```

Comment:
Should this function be renamed to `parseHighlightedLines`?

Reply:
Yes, you're right.

Reply:
Done in 1b192c9.